.github/workflows/schedule.py

```diff
@@ -148,13 +152,15 @@ def _is_eligible(cls, start: str, period: str, now: datetime):
 
     def create_issue(self, title_date: date) -> None:
         title = self.properties['title'] + ' ' + str(title_date)
-        flags = []
+        repo = []
         repository = self.properties.get('_repository')
         if repository is not None:
-            flags.append(f'--repo={repository}')
+            msg = 'The front-matter field `_repository` cannot be empty if specified'
+            reject(repository in ('', '""', "''"), msg)
```

Comment:
You're overthinking this. There are countless more invalid values but you are only rejecting three so this is a drop in a bucket. Keep it simple.